package.json

```diff
@@ -4,8 +4,29 @@
   "description": "Calendar board for PMSIpilot organisation",
```

Comment:
L'app n'est pas assez global pour prétendre être une "Calendar board for Google Apps organisations", je dois virer tous les appels à nos id de calendrier et ajouter ça dans une configuration